### DISCUSSION TITLE
How to hide WWFF column in the advanced view?

### DISCUSSION THREAD

**Discussion by R1BLH** [wavelog/wavelog#581] [General]
It is not possible to hide the WWFF column in the advanced log view. After unchecking the checkbox in the option setting, the column is not removed and the option remains checked.

**Comment by AndreasK79** [comment]
Thanks for letting us know. Pull latest dev and try again.

**Reply by R1BLH** [reply]
> Thanks for letting us know. Pull latest dev and try again.

It's hidden now! Thank you!

One more question about advanced view. Please include column header "Gridsquare" to the translation strings.
![изображение](https://github.com/user-attachments/assets/fe9757df-80a0-4622-aabc-6062cda3bde2)

**Reply by AndreasK79** [reply]
No problem. I translated Contest and Gridsquare. Pull dev now.

**Reply by R1BLH** [reply]
Thank you!
